build-jellyfin.ps1

```diff
@@ -82,14 +86,15 @@ function Install-NSSM {
         [string]$ResolvedInstallLocation,
         [string]$Architecture
     )
+
     Write-Verbose "Checking Architecture"
     if($Architecture -notin @('x86','x64')){
         Write-Warning "No builds available for your selected architecture of $Architecture"
         Write-Warning "NSSM will not be installed"
     }else{
          Write-Verbose "Downloading NSSM"
          # [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-         # Temporary workaround, file is hosted in an azure blob with a custom domain in front for brevity
+         # File is hosted in an azure blob with a custom domain in front for brevity
          Invoke-WebRequest -Uri http://files.evilt.win/nssm/nssm-2.24-101-g897c7ad.zip -UseBasicParsing -OutFile "$tempdir/nssm.zip" | Write-Verbose
```

Comment:
@anthonylavado just thinking out loud - maybe we should include that zip file in this repo?

Reply:
Yeah, we should probably move it in now.

Reply:
Not sure why, given how bad nssm site stability is and that this mirror could also come and go (AFAIR it was introduced by @LogicalPhallacy who's not very active recently).

We can also bump nssm version while we're at it.

Reply:
We can also just throw it on our release website and achieve the same result.

Reply:
Yeah, that also works

Reply:
[done](https://repo.jellyfin.org/releases/other)

Reply:
we can update the link in our next pr